packages/atti-components/src/components/Breadcrumb/Readme.mdx

```diff
@@ -36,52 +34,39 @@ const otherTheme = {
 
 render(
   <ThemeProvider otherTheme={otherTheme}>
-    <Breadcrumb
-      elements={[
-        { name: 'Home', url: '/#' },
-        { name: 'Components', url: '/components' },
-        { name: 'Breadcrumb' },
-      ]}
-      separator={'>'}
-    />
+    <Breadcrumb separator={'>'}>
+        <Breadcrumb.Element url='/#'>Home</Breadcrumb.Element>
+        <Breadcrumb.Element url='/components'>Components</Breadcrumb.Element>
+        <Breadcrumb.Element>Breadcrumb</Breadcrumb.Element>
+    </Breadcrumb>
   </ThemeProvider>,
 )
 ```
-
-The default rendering function expect each element to have at least a name as a string.
-You can also provide an url to generate a link around this element.
-
-You can provide your own rendering function as element.render. You can also pass a global rendering function as renderElement prop.
-These functions will be called from most specific to least specific (ie: element, then global, then default). It'll stop at the first function that return a valid JSX component.
-
+Thanks to the compound component pattern it's up to the parent element to customize the content of each Breadcrumb.Element component.
+Since Breadcrumb is using compound component pattern, renderElement prop is not needed anymore.
+If you want to bring specificity based on props, thanks to use composition in order to generate your custom breadcrumb elements.
 ```js
 const Red = styled(Link)`
   color: red;
 `
 const Green = styled(Link)`
   color: green;
 `
+const CustomBreadCrumbElement = ({children, someProps}) => <Breadcrumb.Element>
+    {someProps === 10 ? <Green>{children}</Green> :  <Red>{children}</Red>}
+ </Breadcrumb.Element>
+
 render(
-  <Breadcrumb
-    elements={[
-      {
-        name: 'Ultra Custom render',
-        someProps: 15,
-        render: element => <Red href="#">{element.name}</Red>,
-      },
-      { name: 'Global Breadcrumb Custom renderer', someProps: 10 },
-      { name: 'Default Renderer with an url', url: '#' },
-      { name: 'Default Renderer without an url' },
-    ]}
-    separator={'>'}
-    renderElement={element => {
-      // Let's render, only when some prop is equals to 10
-      // We return nothing (or null) so that it falls back to the default renderer
-      if (element.someProps === 10) {
-        return <Green href="#">{element.name}</Green>
-      }
-    }}
-  />,
+  <Breadcrumb separator={'>'}>
+
+            <Breadcrumb.Element>
+                <Red href="#">Custom content</Red>
+            </Breadcrumb.Element>
+            <CustomBreadCrumbElement someProps={10}>Custom Breadcrumb.Element</CustomBreadCrumbElement>
+            <CustomBreadCrumbElement someProps={99}>Custom Breadcrumb.Element</CustomBreadCrumbElement>
+            <Breadcrumb.Element url='#'>Default content with an url</Breadcrumb.Element>
+            <Breadcrumb.Element>Default content without an url</Breadcrumb.Element>
+  </Breadcrumb>,
```

Comment:
The syntax formatting of your changes are...special :p Probably want to run prettier on this file

Reply:
Done